# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1457571927 18000
#      Wed Mar 09 20:05:27 2016 -0500
# Node ID 067662ac6bfea2cd70b6f176080e330d5317d499
# Parent  0ad3fe00d6f9083bae3c697b9f706d66e67741ba
Correct typo introduced by changeset 21c3641422e0.

__gnuplot_draw_axes__.m: Replace "dataidx" by "data_idx".

diff --git a/scripts/plot/util/private/__gnuplot_draw_axes__.m b/scripts/plot/util/private/__gnuplot_draw_axes__.m
--- a/scripts/plot/util/private/__gnuplot_draw_axes__.m
+++ b/scripts/plot/util/private/__gnuplot_draw_axes__.m
@@ -1204,33 +1204,33 @@ function __gnuplot_draw_axes__ (h, plot_
               sidx = sidx(2:end);
             else
               style = {};
               sidx = [];
             end
             if (all (obj.facecolor == 1))
               hidden_removal = true;
             else
-              withclause{data_idx} = [withclause{dataidx} " nohidden3d"];
+              withclause{data_idx} = [withclause{data_idx} " nohidden3d"];
             endif
           elseif (facecolor_none_or_white)
             fputs (plot_stream,"set style increment user;\n");
             withclause{data_idx} = sprintf ("with %s %s %d",
                                             style{1}, scmd, sidx(1));
             if (length (style) > 1)
               style = style{2:end};
               sidx = sidx(2:end);
             else
               style = {};
               sidx = [];
             end
             if (all (obj.facecolor == 1))
               hidden_removal = true;
             else
-              withclause{data_idx} = [withclause{dataidx} " nohidden3d"];
+              withclause{data_idx} = [withclause{data_idx} " nohidden3d"];
             endif
           else
             hidden_removal = true;
             withclause{data_idx} = sprintf ("with pm3d linecolor rgb variable");
 
             if (doing_interp_color)
               ## "depthorder" interferes with interpolation of colors.
               dord = "scansautomatic";
